examples/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Examples of using CouchBackup as a library",
   "dependencies": {
     "aws-sdk": "^2.39.0",
-    "couchbackup": "^2.0.0",
```

Comment:
It picked it up for me from node_modules after `npm install ../`. I guess it's probably possible to just require the library directly, but didn't think of it before now.